README-REST.md

```diff
@@ -0,0 +1,1122 @@
+
```

Comment:
I would maybe suggest to move this under `docs/rest.md`. In my view that's a more common approach to nested documentation files. You could also move the logo to that folder.

Reply:
Done